test/fixtures/linter/autofix/ansi/008_looping_rules_l003_l016_l019/before.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
```

Comment:
This is sort of a real-world test for the fix, adapted from the existing file mentioned in the issue, `benchmarks/bench/bench_002_pearson_fix.sql`. 

During the analysis of the issue, three rules were primarily involved in cleaning up the file, and this seemed like a good test case to add to the suite.